app.py

```diff
@@ -13,47 +13,80 @@
+                    "checkbox": {
+                        "equals": True
+                    }
+                }
+            }
+        case "createtime":
+            request_body = {
+                "page_size": 20,
+                "sorts": [
+                    {
+                        "property": "Created time",
+                        "direction": "descending"
+                    }
+                ]
+            }
+
+            if SCAN_FREQUENCY is not None:
+                current_date_time = datetime.datetime.utcnow()
+                timestamp_last_pages_request = current_date_time - \
+                    datetime.timedelta(minutes=(int(SCAN_FREQUENCY) + 1))
+                timestamp_last_pages_request_iso = timestamp_last_pages_request.isoformat()
+                request_body = {
+                    "page_size": 20,
+                    "filter": {
+                        "timestamp": "created_time",
+                        "created_time": {
+                            "after": timestamp_last_pages_request_iso
+                        }
+                    },
+                    "sorts": [
+                        {
+                            "property": "Created time",
+                            "direction": "descending"
+                        }
+                    ]
+                }
+        case _:
+            print(f"Scan method parameter is invalid. Valid values are checkbox or createtime. You have define", SCAN_METHOD)
+            sys.exit()
+    
+    return request_body
+
 
 def read_database(database_id, headers):
+    """
+    It takes a database ID and a set of headers, and returns a list of pages
+    
+    :param database_id: The ID of the database you want to read from
+    :param headers: The headers that we created earlier
+    :return: A list of dictionaries. Each dictionary represents a row in the database.
+    """
     read_url = f"https://api.notion.com/v1/databases/{database_id}/query"
 
-    request_body = {
-        "page_size": 20,
-        "sorts": [
-            {
-                "property": "Created time",
-                "direction": "descending"
-            }
-        ]
-    }
-
-    if SCAN_FREQUENCY is not None:
-        current_date_time = datetime.datetime.utcnow()
-        timestamp_last_pages_request = current_date_time - \
-            datetime.timedelta(minutes=(int(SCAN_FREQUENCY) + 1))
-        timestamp_last_pages_request_iso = timestamp_last_pages_request.isoformat()
-        request_body = {
-            "page_size": 20,
-            "filter": {
-                "timestamp": "created_time",
-                "created_time": {
-                    "after": timestamp_last_pages_request_iso
-                }
-            },
-            "sorts": [
-                {
-                    "property": "Created time",
-                    "direction": "descending"
-                }
-            ]
-        }
+    request_body = get_scan_requestBody()
```

Comment:
Suggesting to make this more 'pythonic' by changing the camelCase into a snake_case (eg get_scan_request_body)